anthem/lyrics/uninstaller.py

```diff
@@ -2,17 +2,9 @@
 # Copyright 2016-2017 Camptocamp SA
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
 
-from ..exceptions import AnthemError
+from . import modules
 
 
 def uninstall(ctx, module_list):
     """ uninstall module """
-    if not module_list:
-        raise AnthemError(u"You have to provide a list of "
-                          "module's name to uninstall")
-
-    mods = ctx.env['ir.module.module'].search([('name', 'in', module_list)])
-    try:
-        mods.button_immediate_uninstall()
-    except:
-        raise AnthemError(u'Cannot uninstall modules. See the logs')
```

Comment:
Could you print a warning using `ctx.log_line`? ("Deprecated: use anthem.lyrics.modules.uninstall instead of anthem.lyrics.uninstaller.uninstall")